tests/test_license_expression.py

```diff
@@ -2219,3 +2226,131 @@ def test_tokenize_or_or(self):
+        result = self.licensing.validate('GPL-2.0-or-later AND MIT')
+        assert result.original_license_expression == 'GPL-2.0-or-later AND MIT'
+        assert result.normalized_license_expression == 'GPL-2.0-or-later AND MIT'
+        assert result.errors == []
+        assert sorted(result.valid_symbols) == sorted(['MIT', 'GPL-2.0-or-later'])
+        assert result.invalid_symbols == []
+        assert result.valid_exception_symbols == []
+
+    def test_validation_invalid_license_key(self):
+        result = self.licensing.validate('cool-license')
+        assert result.original_license_expression == 'cool-license'
+        assert result.normalized_license_expression == 'cool-license'
+        assert result.errors == ['Unknown license key(s): cool-license']
+        assert result.valid_symbols == ['cool-license']
+        assert result.invalid_symbols == ['cool-license']
+        assert result.valid_exception_symbols == []
+
+    def test_validate_exception(self):
+        result = self.licensing.validate('GPL-2.0-or-later WITH WxWindows-exception-3.1')
+        assert result.original_license_expression == 'GPL-2.0-or-later WITH WxWindows-exception-3.1'
+        assert result.normalized_license_expression == 'GPL-2.0-or-later WITH WxWindows-exception-3.1'
+        assert result.errors == []
+        assert result.valid_symbols == ['GPL-2.0-or-later WITH WxWindows-exception-3.1']
+        assert result.invalid_symbols == []
+        assert result.valid_exception_symbols == ['GPL-2.0-or-later WITH WxWindows-exception-3.1']
+
+    def test_validation_exception_with_choice(self):
+        result = self.licensing.validate('GPL-2.0-or-later WITH WxWindows-exception-3.1 OR MIT')
+        assert result.original_license_expression == 'GPL-2.0-or-later WITH WxWindows-exception-3.1 OR MIT'
+        assert result.normalized_license_expression == 'GPL-2.0-or-later WITH WxWindows-exception-3.1 OR MIT'
+        assert result.errors == []
+        assert sorted(result.valid_symbols) == sorted(['GPL-2.0-or-later WITH WxWindows-exception-3.1', 'MIT'])
+        assert result.invalid_symbols == []
+        assert result.valid_exception_symbols == ['GPL-2.0-or-later WITH WxWindows-exception-3.1']
+
+    def test_validation_bad_syntax(self):
+        result = self.licensing.validate('Apache-2.0 + MIT')
+        assert result.original_license_expression == 'Apache-2.0 + MIT'
+        assert result.normalized_license_expression == ''
+        assert result.errors == ['Invalid symbols sequence such as (A B) for token: "+" at position: 11']
+        assert result.valid_symbols == []
+        assert result.invalid_symbols == []
+        assert result.valid_exception_symbols == []
+
+    def test_validation_invalid_license_exception(self):
+        result = self.licensing.validate('Apache-2.0 WITH MIT')
+        assert result.original_license_expression == 'Apache-2.0 WITH MIT'
+        assert result.normalized_license_expression == ''
```

Comment:
The exception symbol is not correct here, but the normalized expression still parses alright and this is fine IMHO
```suggestion
        assert result.normalized_license_expression == 'Apache-2.0 WITH MIT'
```